proxyclient/chainload.py

```diff
@@ -31,6 +32,23 @@
     # Fall back to proxy heap, which will be at the right place in old versions
     new_base = u.memalign(0x10000, memory_size)
 
+def compressed_writemem(iface, dest, data, progress):
```

Comment:
This sounds generically useful, can we move it to utils and make it not leak memory? :-)

Reply:
Good idea! fixed.